app/build.gradle

```diff
@@ -64,6 +64,8 @@ android {
 }
 
 dependencies {
+    implementation project(":screenshots")
```

Comment:
question: If screenshot doesn't match, will CI detect the issue?

Reply:
Yes it should not be the problem. 
I've seen one example GitHub action. At the end of this article https://betterprogramming.pub/sanely-test-your-android-ui-libraries-with-paparazzi-b6d46c55f6b0
![Screenshot 2023-05-16 at 13 31 07](https://github.com/appunite/Loudius/assets/33498031/a2ff5144-f1e2-46ce-93fe-cbe88f53a7b2)

Reply:
Is this already implemented? Have you verified those failures?

Reply:
No, I didn't work on CI yet so I didn't verify how this failure works. But I will take care of CI soon.

Reply:
Np. You can do this later, but IMHO it's a pretty important part of the research.